Due to our new posix-node package, python-wasm does have fork and subprocess support when
run on node.js.  We want to test it fully, since we'll ultimately be using this to do
better builds form source, and we will also bring this to the browser.

diff -Naur native/Lib/test/support/__init__.py wasm/Lib/test/support/__init__.py
--- native/Lib/test/support/__init__.py 2022-08-05 07:45:18.000000000 -0700
+++ wasm/Lib/test/support/__init__.py   2022-08-28 21:06:38.390976006 -0700
@@ -519,10 +519,15 @@

 has_fork_support = hasattr(os, "fork") and not is_emscripten and not is_wasi

+# python-wasm does have fork support, when run on node.js.  We want to test it fully.
+has_fork_support = True
+
 def requires_fork():
     return unittest.skipUnless(has_fork_support, "requires working os.fork()")

 has_subprocess_support = not is_emscripten and not is_wasi
+# python-wasm does have subprocess support, when run on node.js. We want to test it fully.
+has_subprocess_support = True

 def requires_subprocess():
     """Used for subprocess, os.spawn calls, fd inheritance"""

